src/test/java/uk/nhs/digital/nhsconnect/lab/results/model/edifact/RangeDetailTest.java

```diff
@@ -13,7 +13,7 @@
 class RangeDetailTest {
     @Test
     void testGetKey() {
-        assertThat(new RangeDetail(null, null).getKey()).isEqualTo("RND");
+        assertThat(new RangeDetail(null, null, null).getKey()).isEqualTo("RND");
```

Comment:
Minor thing, but test data would look a bit better here if we actually gave proper values, like
`new RangeDetail(new BigDecimal("1"), new BigDecimal("10"), "g/dl")`

Reply:
For `getKey` the values don't matter - which I'm expressing by using null (and empties and zero etc in other classes). If we put values there may be a false impression that they impact the expectation somehow. What do you think?